Corzbank.Validations/ValidateUser.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Corzbank.Data.Entities;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Corzbank.Helpers.Validations
+{
+    public class ValidateUser
```

Comment:
I assume it's better to create just an extension for that, and then use user.IsValid() instead of injection that method

Reply:
i assume inject logger into static class isn't so good idea

Reply:
you don't need loger inside that class

Reply:
can u help with realization IsValid meth, what should be there?